Failure (on Windows) looks related:
```
funqy-amazon-lambda: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
2020-07-01T08:50:37.8102736Z [ERROR] 	[error]: Build step io.quarkus.funqy.deployment.bindings.FunqyScriptGeneratorProcessor#buildScripts threw an exception: java.lang.UnsupportedOperationException
2020-07-01T08:50:37.8103068Z [ERROR] 	at java.base/java.nio.file.Files.setPosixFilePermissions(Files.java:2078)
2020-07-01T08:50:37.8103372Z [ERROR] 	at io.quarkus.amazon.lambda.deployment.LambdaUtil.writeExecutableFile(LambdaUtil.java:64)
2020-07-01T08:50:37.8103669Z [ERROR] 	at io.quarkus.amazon.lambda.deployment.LambdaUtil.generateScripts(LambdaUtil.java:86)
2020-07-01T08:50:37.8103985Z [ERROR] 	at io.quarkus.funqy.deployment.bindings.FunqyScriptGeneratorProcessor.buildScripts(FunqyScriptGeneratorProcessor.java:13)
2020-07-01T08:50:37.8104282Z [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2020-07-01T08:50:37.8104693Z [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2020-07-01T08:50:37.8105012Z [ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2020-07-01T08:50:37.8106327Z [ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2020-07-01T08:50:37.8106657Z [ERROR] 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:932)
2020-07-01T08:50:37.8106950Z [ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
2020-07-01T08:50:37.8107244Z [ERROR] 	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
2020-07-01T08:50:37.8107547Z [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2046)
2020-07-01T08:50:37.8107850Z [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1578)
2020-07-01T08:50:37.8108144Z [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
2020-07-01T08:50:37.8108429Z [ERROR] 	at java.base/java.lang.Thread.run(Thread.java:834)
2020-07-01T08:50:37.8108703Z [ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:479)
2020-07-01T08:50:37.8108962Z [ERROR] -> [Help 1]
```

--- a/extensions/amazon-lambda/common-deployment/src/main/java/io/quarkus/amazon/lambda/deployment/LambdaUtil.java
+++ b/extensions/amazon-lambda/common-deployment/src/main/java/io/quarkus/amazon/lambda/deployment/LambdaUtil.java
@@ -50,6 +50,13 @@ public class LambdaUtil {
         Files.write(artifact, output.getBytes(StandardCharsets.UTF_8));
     }
 
+    public static void writeExecutableFile(OutputTargetBuildItem target, String name, String output) throws IOException {
+        writeFile(target, name, output);
+
+        Path artifact = target.getOutputDirectory().resolve(name);
+        artifact.toFile().setExecutable(true, true);
+    }
+
     public static String copyResource(String resource) throws Exception {
         try (InputStream inputStream = Thread.currentThread().getContextClassLoader().getResourceAsStream(resource)) {
 
@@ -69,14 +76,14 @@ public class LambdaUtil {
 
     public static void generateScripts(String handler, OutputTargetBuildItem target) throws Exception {
         String output = copyResource("lambda/bootstrap-example.sh");
-        writeFile(target, "bootstrap-example.sh", output);
+        writeExecutableFile(target, "bootstrap-example.sh", output);
 
         String lambdaName = artifactToLambda(target.getBaseName());
 
         output = copyResource("lambda/manage.sh")
                 .replace("${handler}", handler)
                 .replace("${lambdaName}", lambdaName);
-        writeFile(target, "manage.sh", output);
+        writeExecutableFile(target, "manage.sh", output);
 
         output = copyResource("lambda/sam.jvm.yaml")
                 .replace("${handler}", handler)

